trillian/integration/demo-script.sh

```diff
@@ -7,6 +7,15 @@ if [ $(uname) == "Darwin" ]; then
 else
   URLOPEN=xdg-open
 fi
+hash ${URLOPEN} 2>/dev/null || { echo >&2 "WARNING: ${URLOPEN} not found - browser windows will fail to open"; }
+if [[ ! -d "${GOPATH}" ]]; then
+  echo "Error: GOPATH not set"
+  exit 1
+fi
+if [[ ${PWD} -ef ${GOPATH}/src/github.com/google/certificate-transparency-go/trillian/integration ]]; then
+  echo "Error: cannot run from directory ${PWD}; try: cd ../..; ./trillian/integration/demo-script.sh"
```

Comment:
I think this doesn't work if GOPATH has multiple entries. Can we just check the file with a relative path? Also this assumes they're 2 levels down, which is likely but may confuse if not. Perhaps refer to the git repository directory in the message?